onnxruntime/python/tools/transformers/fusion_options.py

```diff
@@ -229,8 +228,7 @@ def add_arguments(parser: ArgumentParser):
             required=False,
             action="store_true",
             help="Use MultiHeadAttention instead of Attention operator for testing purpose. "
-            "Note that MultiHeadAttention might be slower than Attention since MatMul of input projection is excluded. "
-            "MultiHeadAttention has only CUDA implementation so the model can only run with cuda execution provider.",
+            "Note that MultiHeadAttention might be slower than Attention since MatMul of input projection is excluded. ",
```

Comment:
Note that MultiHeadAttention might be slower than Attention when qkv inputs are not packed.

Reply:
Fixed